### ISSUE TITLE
[Recipe] Governance

### ISSUE THREAD

**Issue by Jacob-CG** [solana-dev-adv/solana-cookbook#29]
Showing how to use Governance with web3/Rust would be helpful:

- Create a DAO
- Make a Proposal
- Vote on Proposal

**Comment by devyeshtandon** [comment]
Hey @jacobcreech, I would like to take this up. Do we need a seperate section in Recipie, right?

**Comment by jacobcreech** [comment]
@devyeshtandon yes, it would be a separate section. It should use whatever js/ts bindings the governance program has today

**Comment by drprk** [comment]
Hi @jacobcreech does the code snippets need to be necessarily in both js/ts bindings? or will JS do as well?
